cli/schemas/config-file.v1.json

```diff
@@ -698,7 +698,6 @@
           "http",
           "kv",
           "net",
-          "node-globals",
```

Comment:
I think we should keep this for now because this file is still used by the current version of Deno (which is terrible)